docs/RELEASE-NOTES.rst

```diff
@@ -4,24 +4,52 @@ Release Notes for BIG-IP Controller for Kubernetes
 Next Release
```

Comment:
remove `Next Release`. We can have it once 2.0 is released.

Reply:
Review comment incorporated.